bigbluebutton/greenlight
# Recording table front end

## Description

- Recordings tab works now for all recordings from all rooms that belong to the current user
- Can view the table of recordings
- Displays Name, Length, Users, Visibility, and Formats
## Testing Steps

- Pull code
- Create user
- Create room(s) for that user using their user_id
- Create recording(s) using any of those room_id's

Page should show all the recordings you make
## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/38328371/163468562-a40d70db-b198-49b8-b95c-0312c7bcb0b1.png)
